test/support/samples/swagger_formater.ex

```diff
@@ -1,176 +1,248 @@
 defmodule Xcribe.Support.Samples.SwaggerFormater do
-  def path_item_object_without_request_body do
-    %{
-      "put" => %{
-        summary: "",
-        description: "",
-        security: [%{"bearer" => []}],
-        parameters: [
-          %{
-            name: "id",
-            in: "path",
-            required: true,
-            schema: %{type: "string"},
-            example: "590eda29-df5f-4244-8159-7bb87e3cc15a"
-          }
-        ],
-        responses: %{
-          404 => %{
-            description: "",
-            content: %{
-              "application/json" => %{
-                schema: %{properties: %{"message" => %{type: "string"}}, type: "object"}
+  defmodule PathItemObject do
```

Comment:
:nail_care: Seems a little bit strange to have 2 defmodule in the same file, when they fill this many lines.

Reply:
Removed !!